client/sites-dashboard/components/sites-grid.tsx

```diff
@@ -29,13 +30,21 @@ interface SitesGridProps {
 }
 
 export const SitesGrid = ( { sites, isLoading, className }: SitesGridProps ) => {
+	// TODO just for testing the PR - will be removed.
+	const params = new URLSearchParams( window.location.search );
+	if ( params.get( 'sitecount' ) ) {
+		sites = sites.slice( 0, Number( params.get( 'sitecount' ) ) );
+	}
+	const { getBanner } = useLinkInBioBanner();
+	const banner = getBanner( sites, 'grid' );
```

Comment:
The `useLinkInBioBanner` hook is only ever being used to render elements. To me it feels like it should be a component rather than a hook i.e.

```typescript
<LinkInBioBanner sites={ sites } displayMode="grid" />
```

I see the hook also has some shared functions, but I think those can be handled in another way.

Reply:
Yes, good point. That will be cleaner.